code/modules/projectiles/guns/throw/crossbow.dm

```diff
@@ -29,41 +21,48 @@
 	if(cell && severity)
 		emp_act(severity)
 
-/obj/item/gun/throw/crossbow/update_icon()
-	if(!tension)
-		if(!to_launch)
-			icon_state = "[initial(icon_state)]"
-		else
-			icon_state = "[initial(icon_state)]-nocked"
-	else
-		icon_state = "[initial(icon_state)]-drawn"
+obj/item/gun/throw/crossbow/update_icon()
+	icon_state = "[initial(icon_state)]"
+	if(cell)
+		icon_state += "-Cell"
+	if(to_launch)
+		icon_state += "-Loaded"
+	if(tension)
+		icon_state += "-Charged"
+	if(cell.charge >= 500 && tension)
+		icon_state +="-Hot"
 
 /obj/item/gun/throw/crossbow/examine(mob/user)
 	. = ..()
 	if(cell)
 		. += "<span class='notice'>\A [cell] is mounted onto [src]. Battery cell charge: [cell.charge]/[cell.maxcharge]"
+		update_icon()
 	else
 		. += "<span class='notice'>It has an empty mount for a battery cell.</span>"
 
-/obj/item/gun/throw/crossbow/modify_projectile(obj/item/I, on_chamber = 0)
-	if(cell && on_chamber && istype(I, /obj/item/arrow/rod))
+/obj/item/gun/throw/crossbow/modify_projectile(obj/I,)
+	if( tension && (cell.charge >= 1999))
 		var/obj/item/arrow/rod/R = I
 		visible_message("<span class='danger'>[R] plinks and crackles as it begins to glow red-hot.</span>")
-		R.throwforce = 15
+		R.throwforce = 40
 		R.superheated = 1
-		cell.use(500)
+		R.embed_chance = 100
+		R.embedded_fall_chance = 0
+		R.embedded_pain_multiplier = 3
+		R.sharp = 1
+		cell.use(2000)
 
 /obj/item/gun/throw/crossbow/get_throwspeed()
-	return tension * speed_multiplier
+	return 5 * speed_multiplier
 
 /obj/item/gun/throw/crossbow/get_throwrange()
-	return tension * range_multiplier
+	return 5 * range_multiplier
 
 /obj/item/gun/throw/crossbow/process_chamber()
 	..()
 	if(to_launch)
 		modify_projectile(to_launch, 1)
-	update_icon()
+	  update_icon()
```

Comment:
There's some extra whitespace here which is why this isn't compiling. You only want one tab here, no spaces